src/plays/dev-jokes/devJokes.css

```diff
@@ -0,0 +1,76 @@
+.maincontanier {
+  display: flex;
+  justify-content: center;
+  flex-direction: column;
+  align-items: center;
+  background-image: url("background.jpg");
+  background-position: center;
+  background-size: cover;
+  height: 100%;
+  width: 100%;
+}
+.jokecontanier {
+  margin: 20px;
```

Comment:
The parent container is a flex box, do you need a margin too? Specially when you have a margin for button too